### PR TITLE
Feature/cplp 1515/app detail page

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**

### CODE REVIEW

File: cx-portal-shared-components/src/components/content/Cards/CardDecision.tsx [hunk 1]
```diff
@@ -34,14 +34,22 @@ export interface CardDecisionProps {
   items: AppContent[]
   onApprove: any
   onDelete: any
+  onClick: any
```
Comment (manojava-gk) [inline cx-portal-shared-components/src/components/content/Cards/CardDecision.tsx:37]: Could you specify the type here?

Comment (nidhigarg-bmw) [reply cx-portal-shared-components/src/components/content/Cards/CardDecision.tsx:37]: We are using **any** for all click events where we are sending data using callback.

Comment (manojava-gk) [reply cx-portal-shared-components/src/components/content/Cards/CardDecision.tsx]: @nidhigarg-bmw  has updated this. Marking as resolved

**Review by manojava-gk — APPROVED** [review]
lftm

**Pull request merged**